misago/users/tests/test_djangoadmin_user.py

```diff
@@ -1,47 +1,64 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import Permission
 from django.core.urlresolvers import reverse
-from django.test import override_settings
 from django.utils import formats
+from django.utils.translation import ugettext as _
+from django.test import override_settings
 
 from misago.admin.testutils import AdminTestCase
 
 
 @override_settings(ROOT_URLCONF='misago.core.testproject.urls')
 class TestDjangoAdminUserForm(AdminTestCase):
-    def test_edit_page_content(self):
-        """assert that edit-view of `test_user` contains expected content."""
-        User = get_user_model()
-
-        test_user = User.objects.create_user(
+    user = None
+    user_list_view_url = None
+    user_edit_view_url = None
+    
+    def setUp(self):
+        super(TestDjangoAdminUserForm, self).setUp()
+        self.user = get_user_model().objects.create_user(
             username='Bob',
             email='bob@test.com',
             password='Pass.123',
         )
-
-        test_user_edit_view = reverse(
+        self.user_list_view_url = reverse('admin:misago_users_user_changelist')
+        self.user_edit_view_url = reverse(
             viewname='admin:misago_users_user_change',
-            args=[test_user.id],
+            args=[self.user.id],
         )
-        response = self.client.get(test_user_edit_view)
 
-        self.assertEqual(response.status_code, 200)
+    def test_user_edit_view_content(self):
```

Comment:
Well, it doesn't assert that the user is editable, because there is no POST request there.

The first thing that comes to my mind is `Check edit view content and presence of the link to misago admin panel.` Looks like the first part of the docstring like that will only repeat the test name, and the second... I suppose the word `and` shows that the test can be splitted to `test_user_edit_view_content` and `test_misago_admin_url_presence_in_user_edit_view`. Yet then I can't came up with any meaningful docstrings for them anyway, because the names seems self-explainable, what do you think?